QuizApp/QuizApp/UC/Quiz/QuizUseCase.swift

```diff
@@ -2,6 +2,8 @@ protocol QuizUseCaseProtocol {
 
     func getQuizes(for category: QuizCategoryModel) async throws -> [QuizModel]
 
+    func getQuizes() async throws -> [QuizModel]
```

Comment:
Functions that return a value and have no arguments should be computed properties.